x-pack/plugins/monitoring/server/lib/metrics/apm/classes.ts

```diff
@@ -5,12 +5,16 @@
  * 2.0.
  */
 
+/* eslint-disable max-classes-per-file */
+
+import { i18n } from '@kbn/i18n';
+// @ts-ignore
```

Comment:
Is the plan to come back and remove the ignores in another PR?

Reply:
Yes, that's part of https://github.com/elastic/kibana/issues/106761